src/Backend/Modules/Location/Engine/Model.php

```diff
@@ -217,6 +217,7 @@ public static function insert(array $item): int
         // insert extra
         $item['extra_id'] = BackendModel::insertExtra(
             ModuleExtraType::widget(),
+            'Location',
```

Comment:
This won't fix the already installed modules when upgrading, can you also update the config of the module so the default isn't index but has the correct value of Location?